app/src/store/modules/explorer/curation/actions.js

```diff
@@ -88,5 +89,61 @@ export default {
 
     const response = await fetchResponse.json()
     return { response, identifier: chartInstanceObject.identifier }
+  },
+
+  async lookupOrcid ({ commit }, orcidId) {
+    const unhyphenated = /^\d{15}(\d|X)$/.test(orcidId)
+    unhyphenated && (orcidId = orcidId.replace(/^\(?(\d{4})\)?(\d{4})?(\d{4})?(\d{3}(\d|X))$/, '$1-$2-$3-$4'))
+
+    if (isValidOrcid(orcidId)) {
+      // TODO: update the endpoint route name
+      const url = `/api/knowledge/images?uri=http://orcid.org/${orcidId}&view=describe`
+      const response = await fetch(url, {
+        method: 'GET'
+      })
+      if (!response || response?.statusText !== 'OK') {
+        const snackbar = {
+          message: response.message || 'Something went wrong while fetching orcid data',
+          duration: 5000
+        }
+        return commit('setSnackbar', snackbar, { root: true })
+      }
+
+      if (response.status === 201) {
+        return
+      }
```

Comment:
Should this be here? Can we use the response as is and optionally chain after

Reply:
Yes, I'll try this. I think I was basing this off of the `loadItems` action in gallery/actions.js